cmd/gitops/install/cmd.go

```diff
@@ -123,10 +121,12 @@ func installRunCmd(cmd *cobra.Command, args []string) error {
 
 		gitClient, gitProvider, err = factory.GetGitClients(context.Background(), providerClient, services.GitConfigParams{
 			// We need to set URL and ConfigRepo to the same value so a deploy key is created for public config repos
-			URL:        installParams.ConfigRepo,
-			ConfigRepo: installParams.ConfigRepo,
-			Namespace:  namespace,
-			DryRun:     installParams.DryRun,
+			URL:               gitopsParams.ConfigRepo,
+			ConfigRepo:        gitopsParams.ConfigRepo,
+			Namespace:         namespace,
+			DryRun:            gitopsParams.DryRun,
+			FluxHTTPSUsername: gitopsParams.FluxHTTPSUsername,
+			FluxHTTPSPassword: gitopsParams.FluxHTTPSPassword,
```

Comment:
In my branch to address the conflicts, I had shifted this to passing in a `*flux.HTTPSCreds` rather than the two fields.